Sensitive and confidential info in Working Open (#79)
hyphacoop/handbook: working-open.md

# How and what?

- We currently accommodate redactions and going off-the-record for meeting notes. Participants can request to stop taking notes, or redact and correct notes after they have been taken.

> - We recognize that some information should be treated as sensitive and confidential for the benefit of Hypha and its clients and partners. An employee or contractor with access to such information is responsible for keeping this information confidential.

Reviewer: What is a "partner" here? think we're following out of sync with previous preferred term "collaborator", we should update either this page, or this one: https://handbook.hypha.coop/co-operative.html#about-us

Author: Sticking with what's existing.

```suggestion
- We recognize that some information should be treated as sensitive and confidential for the benefit of Hypha and its clients and collaborators. An employee or contractor with access to such information is responsible for keeping this information confidential.
```

Author: Nevermind that, see below.

> - We recognize that some information should be treated as sensitive and confidential for the benefit of Hypha and its clients and partners. An employee or contractor with access to such information is responsible for keeping this information confidential.

Reviewer: ```suggestion
- We recognize that some information should be treated as sensitive and confidential for the benefit of Hypha and its clients and partners. Anyone with access to confidential information is responsible for doing their best to keep confidential information secret and expected to adhere to these guidelines.
```
Mirroring language from our independent contractor agreement and IP assignment.

Also I think you may want to use the concept/explain what PII is here if you mention it below.

Reviewer: What is personal information?

Under PIPEDA, personal information includes any factual or subjective information, recorded or not, about an identifiable individual. This includes information in any form, such as:

    age, name, ID numbers, income, ethnic origin, or blood type;
    opinions, evaluations, comments, social status, or disciplinary actions; and
    employee files, credit records, loan records, medical records, existence of a dispute between a consumer and a merchant, intentions (for example, to acquire goods or services, or change jobs).

https://www.priv.gc.ca/en/privacy-topics/privacy-laws-in-canada/the-personal-information-protection-and-electronic-documents-act-pipeda/pipeda_brief/#_h2

Author: ```suggestion
- We recognize that some information should be treated as sensitive and confidential for the benefit of Hypha and its clients and collaborators. Anyone with access to confidential information is responsible for doing their best to keep that information secret and are expected to adhere to these guidelines.
```

**Default closed to specific working groups and project members**:

- Scope: Organization

>   - Full personal/financial information of members (e.g., home addresses, banking information, and social insurance numbers) 📝 

Reviewer: ```suggestion
  - Full financial and personal information of members (e.g., home addresses, banking information, and social insurance numbers) 📝 
```

- Chat transcripts
- Notes involving external parties and collaborators
- Materials from project retrospectives

**Sensitive and confidential**:

- Passwords, credentials, and sensitive personal information
- Data sets, software, and other content in project work that we don't have consent to share

> - Financial or other information of our clients and partners that we don't have consent to share

Reviewer: ```suggestion
- Financial information of our clients and partners
```
not sure what "other information" is here, if not covered by above line?

Reviewer: Flagging term partner here too needs to be updated

Author: other information -> anything that we do not have consent to share, whether or not of financial or personal nature

```suggestion
- Financial or other information of our clients and collaborators that we don't have consent to share
```

Reviewer: What would that be except "other content in project work" or the things identified above? "other business information"

Author: ```suggestion
- Financial or other business information of our clients and collaborators that we don't have consent to share
```

> - Data collected through our digital infrastructure, which may contain user account information or personally identifiable information (PII)

Reviewer: ```suggestion
- Data collected through our digital infrastructure, which may contain user account information or personal information
```
EDIT: suggest just using personal information to be consistent, and use the PIPEDA definition I put above, which doesn't have PII

Author: I think this is fine, and we do not need to link to PIPEDA or expand on it.
